packages/perennial-vaults/test/integration/helpers/setupHelpers.ts

```diff
@@ -0,0 +1,62 @@
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers'
+import { BigNumberish, utils } from 'ethers'
+import { createPayoffDefinition } from '../../../../common/testutil/types'
+import { ChainlinkOracle__factory, IController__factory, IProduct } from '../../../types/generated'
+import { JumpRateUtilizationCurveStruct } from '../../../types/generated/@equilibria/perennial/contracts/interfaces/IProduct'
+
+export interface DeployProductParams {
```

Comment:
Can we import the `IProduct.ProductInfo` and use typescript's `Partial<..>` below?